### DISCUSSION TITLE
[Documentation] Nearly all Chinese (Simplified) documentation translations are in poor quality

### DISCUSSION THREAD

**Discussion by GalvinGao** [vuetifyjs/vuetify#12643] [General]
### Environment

**Browsers:** Chrome 86.0.4240.198
**OS:** Mac OS 11.0.0

### Steps to reproduce
For reproduction or the significant of how's the quality is, one may visit the following URLs:

#### Page [Treeshaking](https://vuetifyjs.com/zh-Hans/features/treeshaking/)
As treeshake is a term commonly used in the JavaScript context for dead-code elimination ([ref](https://webpack.js.org/guides/tree-shaking/)), directly translating those words into Chinese may cause confusion and difficulty when trying to navigate to a specific page. In particularly this case, `tree``shaking` has been separately translated to dedicated characters **without the original word being kept**.

Also in this page, inconsistent usage of spaces occurs between `从lib导入` and `Vue 配置安装`. Those inconsistent usage of spaces oftenly occurs when using **Machine Translations**.

Apart from that, quality for those particular two terms are also poor. `从lib导入` might translate `importing` as `引入`, and in `Vue 配置安装` the order of language has messed and it doesn't point out the `vue config` as the english doc does. Ideally this translation could be `使用 Vue.config.js 配置`.

### Expected Behavior
`Treeshaking`: (left as-is, or `树摇 (Treeshaking)`)
`Importing from lib`: `从 lib 引入`
`Vue config installation`: `使用 Vue.config.js 配置`

### Actual Behavior
`Treeshaking`: `树摇`
`Importing from lib`: `从lib导入`
`Vue config installation`: `Vue 配置安装`

### Reproduction Link
<a href="https://vuetifyjs.com/zh-Hans/features/treeshaking/" target="_blank">https://vuetifyjs.com/zh-Hans/features/treeshaking/</a>

### Other comments
The point of this issue is not only pointing out those particular poor quality translations, but to emphasis this problem to the scope of all Vuetify documentation translations. I'm not saying that we shall avoid machine translations at all, but personally I strongly recommend to avoid directly using the machine translation as it comes with cost of degraded quality. I'm really happy to see that Chinese (Simplified) has been listed as star on Crowdin and I'm both proud and much appreciate the Vuetify community has chose Chinese as one of the languages to support with, but I still want to say that the quality is always more important than the actual nomination of the language. Currently, a large number of Vuetify documentation translated in Chinese (Simplified) are in machine translations, in my opinion, and shall be temporarily taken down for polish and rewrite.

Personally, I'm more than happy to improve the Vuetify documentation quality. To accomplish this, I also propose to come with a Glossary as it seems to lack from the Vuetify Crowdin project.

Please forgive for my rudeness if it occurs in any part of my discussion as English is my second language :D

Thanks in advance for the consideration from Vuetify! :D

**Comment by MajesticPotatoe** [comment]
All translations are handled by the community, and entirely done via Crowdin. This is better off moved to a discussion. As issues are for reporting bugs.

It is always possible someone went through and machine translated everything. But its also possible that most of that came in from the old docs, (or a mixture of both).

It may even be beneficial to make a process in which translations have to be approved before they are added to the build, however that comes with significant overhead (people to manage, and have time to manage, champions of the given language). We welcome anyone to come and translate, so If you have the time to do so, it is always appreciated.

**Reply by GalvinGao** [reply]
Sorry for the late reply here as GitHub doesn't seems to give me any notifications for both your reply or the transfer of the issue :(

I would be happy to contribute with translations, but before that I think to build up an official Glossary that we could reference with in the future might be more essential. If that's suitable then I could help and construct one first :D
